pkg/arm/v8/resources.go

```diff
@@ -441,6 +442,12 @@ func vmss(cs *api.OpenShiftManagedCluster, app *api.AgentPoolProfile, backupBlob
 						},
 					},
 				},
+				DiagnosticsProfile: &compute.DiagnosticsProfile{
```

Comment:
Relevant SDK docs:

- https://godoc.org/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute#VirtualMachineScaleSetVMProfile
- https://godoc.org/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute#BootDiagnostics

This LGTM, the only thing I'm not 100% sure of is whether this is the right storage account. @mjudeikis can you TAL?